OneLogin SSO documentation updates (#650)
rudderlabs/rudderstack-docs: docs/user-guides/administrators-guide/onelogin-sso.mdx

<img src="../../assets/user-guides/onelogin-1.png" alt="Administration option in OneLogin" />

2. From the top menu, go to **Applications** > **Applications**, as shown:

<img src="../../assets/user-guides/onelogin-2.png" alt="Applications option" />

3. Then, click **Add App**, as shown:

<img src="../../assets/user-guides/onelogin-3.png" alt="Add App option" />

4. In the resulting **Find Applications** page, search for **SAML Custom Connector (Advanced)**. From the search results, select the application, as shown:

<img src="../../assets/user-guides/onelogin-4.png" alt="Select SAML Custom Connector option" />

5. Name your SAML app and click on **Save**, as shown:

<img src="../../assets/user-guides/onelogin-5.png" alt="Select SAML app name" />

6. In the **Configuration** tab, enter the settings as shown in the following image:

<img src="../../assets/user-guides/onelogin-6.png" alt="SAML app configuration" />

The settings to be configured are listed in the following table:

| Setting | Value |
| :-------| :------|
| Audience (EntityID) | `urn:amazon:cognito:sp:us-east-1_ABZiTjXia` |
| Recipient | `https://auth2.rudderstack.com/saml2/idpresponse` |

> | ACS (Consumer) URL Validator | `^https:VVauth2\.rudderstack\.comVsamI2VidpresponseV\S` |

Reviewer: ```suggestion
| ACS (Consumer) URL Validator | `^https:\/\/auth2\.rudderstack\.com\/saml2\/idpresponse\/\$` |
```

Reviewer: This is a regex pattern and some characters need to be escaped

</div>

7. From the dropdown, select the **SAML initiator** and **SAML nameID format** fields as shown:

<img src="../../assets/user-guides/onelogin-7.png" alt="SAML settings" />

<div class="successBlock">

Configure the other SAML settings related to the assertion validity, encryption method, etc. as per your organizational requirements.
</div>

8. Next, go to the **Parameters** tab and add the custom parameters as shown below:

<img src="../../assets/user-guides/onelogin-8.png" alt="Custom parameters" />

The custom parameters and their values are listed in the following table:

| Parameter | Value |
| :-------| :------|
| Email | `Email` |
| LastName | `Name` |
| NameID value | `Email` |

<div class="infoBlock">

For the <strong>LastName</strong> custom attribute, you can specify a single field <code class="inline-code">Name</code> - which specifies how you would like to see your employees on the RudderStack web app.
</div>

> 9. To add a custom parameter, simply click the **+** button, enter the **Field name**, and select the value from the dropdown, as shown:

Reviewer 2: ```suggestion
9. To add any other custom parameter, click the **+** button, enter the **Field name**, and select the value from the dropdown, as shown:
```
